src/test/fakeSuite/Repros/.vscode/preconfigure.bat

```diff
@@ -0,0 +1,3 @@
+@echo off
+set PATH="%cd%\bin\InterestingSmallMakefile\CL\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\12.34.56789\bin\Hostx86\x86";%PATH%
```

Comment:
does it have to be such a long path, or can it just be `bin\windows\compilers` or something like that?

Reply:
I only wanted to keep as close to a real VS compiler path. Also, there are a few characteristics that are needed, where parsing could go wrong. I need the space " ", I need "c" after a "\"...etc.... Let's leave it as is for now.